cypress/integration/person.js

```diff
@@ -30,3 +30,32 @@ describe('Person Show All Distinct Checks', () => {
         cy.get('[data-cy="show-more-distinct-id"]').should('not.exist')
     })
 })
+
+describe('Merge person', () => {
+    beforeEach(() => {
+        cy.clickNavMenu('persons')
+        cy.get('[data-attr=persons-search]').type('deb').should('have.value', 'deb')
+        cy.get('.ant-input-search-button').click()
+        cy.contains('deborah.fernandez@gmail.com').click()
+    })
+
+    it('Should merge person', () => {
+        cy.get('.extra-ids').should('not.exist') // No extra IDs
+        cy.contains('$create_alias').should('not.exist')
+        cy.get('span:contains(Pageview)').should('have.length', 1)
+        cy.get('span:contains(clicked)').should('have.length', 1)
+
+        // Merge people
+        cy.get('[data-attr=merge-person-button]').click()
+        cy.get('.ant-select-multiple').type('amalia')
+        cy.contains('amalia.potts@hotmail.com').click()
+        cy.contains('OK').click()
+
+        cy.wait(2000)
```

Comment:
This is quite optimistic on github actions settings.

Increasing the timeout just slows down tests as well.

A better solution might be to reload-and-check until the below condition is true UP TO e.g. 20s. https://docs.cypress.io/guides/core-concepts/retry-ability

See https://github.com/PostHog/posthog-js/blob/master/cypress/support/commands.js#L44-L49 for a custom command which retries until it succeeds. 


----

Aside - we don't reset the db between tests right? In that case couldn't this start causing test-ordering related issues?

Reply:
> Aside - we don't reset the db between tests right? In that case couldn't this start causing test-ordering related issues?

It's a good point, I would have thought so too but tests seem to have passed fine?

Can configure this to be more general

Reply:
It might be fine as long as these users aren't directly or indirectly asserted against in other tests :)